source/libraries/ContractDeployer.ts

```diff
@@ -19,6 +19,7 @@ export class ContractDeployer {
     public readonly contracts = [];
     public readonly signatures = [];
     public readonly bytecodes = [];
+    public readonly transactionReceipts = [];
```

Comment:
This doesn't appear to be used anywhere, should be deleted.

Reply:
I was using this to debug so I could look at the receipts just before creating the new market.  I'll remove it.